### ISSUE TITLE
Don't print gpu-func compile messages to *standard-out*, use something quieter

### ISSUE THREAD

**Issue by cbaggers** [cbaggers/cepl#155]
I'm not sure what the correct approach is for this. I'll ask on IRC later

**Comment by stacksmith** [comment]
*error-output* is a sensible destination for error messages.

**Comment by cbaggers** [comment]
True, but this is about notes from the compiler. Sbcl prints some ancillary info for certain actions (like when compiling a function) I think this is outside the spec
